libraries/Bluefruit52Lib/src/bluefruit.cpp

```diff
@@ -676,6 +685,8 @@ void SD_EVT_IRQHandler(void)
   // Notify both BLE & SOC Task
   xSemaphoreGiveFromISR(Bluefruit._soc_event_sem, NULL);
   xSemaphoreGiveFromISR(Bluefruit._ble_event_sem, NULL);
+  // Notify parallel multiprotocol Task, if any
+  if (Bluefruit._mprot_event_sem!=NULL) xSemaphoreGiveFromISR(*Bluefruit._mprot_event_sem, NULL);
```

Comment:
please wrap all the ant related code inside `#ifdef ANT_LICENSE_KEY` there must be zero code addition when ANT key is not defined.

Reply:
Ok, I´ll do that soon